src/main.rs

```diff
@@ -51,11 +52,39 @@ use std::fs::File;
 use std::io::BufReader;
 use std::io::Read;
 use std::path::Path;
+use tss_esapi::constants::algorithm::AsymmetricAlgorithm;
+use uuid::Uuid;
 
 use error::{Error, Result};
 
 static NOTFOUND: &[u8] = b"Not Found";
 
+fn get_uuid(agent_uuid_config: &str) -> String {
+    match agent_uuid_config {
```

Comment:
Will this cover all possible entries or should there be a default match as well?

Reply:
@ashcrow Thanks for looking at this - line 77 below is supposed to take care of any cases that don't match these pre-defined strings. Were you thinking of something else?

Reply:
Ah I see. The submatch attempts to parse and either passes and returns or fails and returns a new uuid. That should be fine. My brain was looking for a `_` match :-)

Reply:
Ah yeah, gotcha. : ) Thanks for double checking.